src/lib/wallets/connecters/bitGet/bitGetWallet.ts

```diff
@@ -0,0 +1,148 @@
+import type { Connector } from "wagmi/connectors";
+import type { InjectedConnectorOptions } from "@wagmi/core/connectors/injected";
+import { InjectedConnector } from "wagmi/connectors/injected";
+import { Chain, Wallet, getWalletConnectConnector } from "@rainbow-me/rainbowkit";
+
+export interface BitKeepWalletLegacyOptions {
+  projectId?: string;
+  chains: Chain[];
+  walletConnectVersion: "1";
+  walletConnectOptions?: any;
+}
+
+export interface BitKeepWalletOptions {
+  projectId: string;
+  chains: Chain[];
+  walletConnectVersion?: "2";
+  walletConnectOptions?: any;
+}
+
+export async function getWalletConnectUri(connector: Connector, version: "1" | "2"): Promise<string> {
+  const provider = await connector.getProvider();
+  return version === "2"
+    ? new Promise<string>((resolve) => provider.once("display_uri", resolve))
+    : provider.connector.uri;
+}
+
+export function isAndroid(): boolean {
+  return (
+    typeof navigator !== "undefined" && /Android\s([0-9.]+)/.test(navigator.userAgent) // Source: https://github.com/DamonOehlman/detect-browser/blob/master/src/index.ts
+  );
+}
+
+export const bitGetWallet = ({
+  chains,
+  projectId,
+  walletConnectOptions,
+  walletConnectVersion = "2",
+  ...options
+}: (BitKeepWalletLegacyOptions | BitKeepWalletOptions) & InjectedConnectorOptions): Wallet => {
+  const isBitKeepInjected =
+    typeof window !== "undefined" &&
+    // @ts-expect-error
+    window.bitkeep !== undefined &&
+    // @ts-expect-error
+    window.bitkeep.ethereum !== undefined &&
+    // @ts-expect-error
+    window.bitkeep.ethereum.isBitKeep === true;
+
+  const shouldUseWalletConnect = !isBitKeepInjected;
+
+  return {
+    id: "bitKeep",
+    name: "Bitget Wallet",
+    iconUrl: async () => (await import("./bitGetWallet.svg")).default,
```

Comment:
why we don't use importImage here?

Reply:
I am just trying to follow the Rainbowkit adapter modal so it will be easier to update and fix.